lib/cionic/emg_iir.c

```diff
@@ -15,6 +15,10 @@ float emg_lowpass_filter_sos[NO_OF_BQS][NO_OF_COEFFS_PER_BQ] = {
     {1.0,2.0,1.0,1.0,-1.4083885078365557,0.7369080185138834},
 };
 
+uint8_t emg_sub_sampling_rate = 1;   
+uint16_t ma_size = EMG_RMS_MA_SIZE;   
```

Comment:
Would be cleaner to define ma_size as static. Also could you add g_ in front of the variable name to refer it as global to the file?

Reply:
done